Dockerfile

```diff
@@ -1 +1,88 @@
+# Set up ARG variable to allow the user to specify the tag version at build time
+ARG UBUNTU_VERSION=22.04
```

Comment:
On this ARG UBUNTU_VERSION, we are suppose to call this argument at run time, providing the OS version, therefore I believe we shouldn't specified the OS version on the Dockerfile.